src/interfaces/payment-interfaces/taxes-interface.ts

```diff
@@ -0,0 +1,5 @@
+/** Interface for Fees Taxes from Payment API. */
+export interface TaxesIF {
```

Comment:
If you wanted, since taxes are intimately tied to fees, you could declare this interface in the same file as fees. Up to you.

Reply:
👍